Mathlib/Data/Polynomial/Eval.lean

```diff
@@ -418,7 +418,7 @@ theorem eval_smul [Monoid S] [DistribMulAction S R] [IsScalarTower S R R] (s : S
 
 @[simp]
 theorem eval_C_mul : (C a * p).eval x = a * p.eval x := by
-  -- Porting note: `apply` → `induction`
+  -- Porting note (#11443): `apply` → `induction`
```

Comment:
Let's just drop these comments, the new `induction` is better than `apply` anyway.

Reply:
Ok @eric-wieser, I'm going to remove all these porting notes then.

Reply:
Done.